### PR TITLE
Changed the default monitor branch to branch-3.0

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
and adjusted the monitor node's installtion and configuration to match the new branch

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
~~- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)~~
~~- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] All new and existing unit tests passed (CI)
~~- [ ] I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3329,7 +3329,7 @@ def __init__(self, targets, params):
         self.local_metrics_addr = start_metrics_server()  # start prometheus metrics server locally and return local ip
         self.sct_ip_port = self.set_local_sct_ip()
         self.grafana_port = 3000
-        self.monitor_branch = self.params.get('monitor_branch', default='branch-2.4')
+        self.monitor_branch = self.params.get('monitor_branch', default='branch-3.0')
```
Comment (roydahan) [inline sdcm/cluster.py]: why do we need another default other than the test_default?

File: sdcm/cluster.py [hunk 2]
```diff
@@ -3586,11 +3589,13 @@ def reconfigure_scylla_monitoring(self):
             configure_script = dedent("""
                         cd {0.monitor_install_path}
                         mkdir -p {0.monitoring_conf_dir}
-                        ./genconfig.py -s -n -d {0.monitoring_conf_dir} {0._monitoring_targets}
+                        python 3.6 genconfig.py -s -n -d {0.monitoring_conf_dir} {0._monitoring_targets}
```
Comment (roydahan) [inline sdcm/cluster.py]: I think there is unneeded  space here between python and 3.6

File: sdcm/cluster.py (hunk 1 above)
Comment (bentsi) [reply sdcm/cluster.py]: right Roy, please remove the default value here

File: sdcm/cluster.py [hunk 3]
```diff
@@ -3440,10 +3440,13 @@ def wait_for_init(self):
...
             prereqs_script = dedent("""
+                yum install -y wget
+                yum install -y unzip
                 yum install -y epel-release
-                yum install -y python-pip unzip wget
-                pip install --upgrade pip
-                pip install pyyaml
+                yum install -y python36
+                yum install -y python36-pip
+                python36 -m pip install --upgrade pip
```
Comment (bentsi) [inline sdcm/cluster.py]: never run python using specific version binary, if it is python 3 it should be `python3`

File: sdcm/cluster.py [hunk 4]
```diff
@@ -3535,7 +3538,7 @@ def remove_sct_metrics(x):
         configure_script = dedent("""
             cd {0.monitor_install_path}
             mkdir -p {0.monitoring_conf_dir}
-            ./genconfig.py -s -n -d {0.monitoring_conf_dir} {0._monitoring_targets}
+            python3.6 genconfig.py -s -n -d {0.monitoring_conf_dir} {0._monitoring_targets}
```
Comment (bentsi) [inline sdcm/cluster.py]: `python3`

File: sdcm/cluster.py (hunk 1 above)
Comment (ShlomiBalalis) [reply sdcm/cluster.py]: Done and done

File: sdcm/cluster.py (hunk 3 above)
Comment (ShlomiBalalis) [reply sdcm/cluster.py]: changed to python3

File: sdcm/cluster.py (hunk 4 above)
Comment (ShlomiBalalis) [reply sdcm/cluster.py]: changed to python3

**Comment by bentsi** [comment]
please resolve conflicts

**Comment by ShlomiBalalis** [comment]
@bentsi Conflicts solved

**Pull request merged**